### DISCUSSION TITLE
More than one smartbanner

### DISCUSSION THREAD

**Discussion by tomo93** [ain/smartbanner.js#542] [Q&A]
hello, is it possible include two separate smartbanners with this library? Because i have two banner and if I try to  close one banner, even the other will close. Can you help me @ain ? :)

**Comment by ain** [comment]
Just to clarify the motivation, is your goal to promote two different apps at once on a single page?

**Reply by tomo93** [reply]
I have a single page application, but i want one banner in one page and another on a different page

**Reply by ain** [reply]
How do you deal with it currently? 

Leaving the same `meta` tags on all pages usually leads to SEO problems. For that purpose it's advised to use dynamic page-specific meta tags, esp. title. This would also work around your issue.

**Reply by ain** [reply]
There's #589 in the works with much of what is needed done already. It'd also solve the above-described use-case.
